### PR TITLE
[INFRA-104]:feat/add-script-to-delete-shards-without-contracts

### PR DESCRIPTION
**Pull request by fabioespinosa**
Iterates over Shards, if the contracts property doesn't exist, or the array has length 0. It will delete the shard from Mongo DB.

### CODE REVIEW

File: bin/delete-shards-without-contracts.js [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+      }
+    });
+
+    let idShardBeingChecked;
+
+    let deleteCount = 0;
+    let checkedShards = 0;
+
+    const checkAndDeleteShard = (shard, cb) => {
+      checkedShards += 1;
```
Comment (sg-gs) [inline bin/delete-shards-without-contracts.js]: Is a great idea to keep the count of checked and deleted count but try to do it outside this reusable method, therefore you will avoid problems like: "I have to call this method but I do not want to increase these counters" (in a retry for example)

File: bin/delete-shards-without-contracts.js [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+          return;
+        }
+        console.log('finished');
+        logStatus();
+        stopScript();
+      });
+    });
+  } catch (err) {
+    console.error('Unexpected error');
+    console.error(err.message);
```
Comment (sg-gs) [inline bin/delete-shards-without-contracts.js]: These logs could be merged in once: `console.log("Unexpected error: %s", err.message);`

**Review by sg-gs — CHANGES_REQUESTED** [review]

File: bin/delete-shards-without-contracts.js (hunk 1 above)
Comment (fabioespinosa) [reply bin/delete-shards-without-contracts.js]: good idea, fixed

File: bin/delete-shards-without-contracts.js (hunk 2 above)
Comment (fabioespinosa) [reply bin/delete-shards-without-contracts.js]: fixed

**Pull request closed without merging**